dali-mqtt-daemon.py

```diff
@@ -63,95 +91,83 @@ def gen_ha_config(light):
     return json.dumps(json_config)
 
 log_format = '%(asctime)s %(levelname)s: %(message)s'
-logging.basicConfig(format=log_format, level=logging.INFO)
+logging.basicConfig(format=log_format, level=logging.DEBUG)
 logger = logging.getLogger(__name__)
 
-def dali_scan(dali_driver, max_range=4):
+def dali_scan(driver_object, max_range=4):
```

Comment:
Don't understand the need for the rename

Reply:
It's look like my mistake. I was rename argument name in other functions and by mistake here too